draft-packages/form/KaizenDraft/Form/Primitives/FieldMessage/styles.scss

```diff
@@ -24,6 +25,13 @@ $ca-palette-input-validation-negative: $kz-color-coral-600;
   }
 }
 
+.caution {
+  padding: $ca-grid / 2;
+  color: $kz-color-wisteria-800;
+  border-radius: $message-border-radius;
+  background: $kz-color-yuzu-300;
+}
```

Comment:
Everything except for the `background` is the same as the error styling, so you could DRY this up with something like (and move it below the error styling).
```
.caution {
  composes: error;
  background: $kz-color-yuzu-300;
}
```

Reply:
Also, looking at Figma, I believe this is meant to be Yuzu 200 instead of 300